docs/agent/config.md

```diff
@@ -8,6 +8,14 @@ might mean:
  * The option refers to a feature that's currently under development
  * The option refers to a feature that's scheduled but will come later
 
+## Environment variables
+
+All options supported by the Infrastructure Agent in the configuration file can also be set through environment variables;
+ * Option names should be put in uppercase with the `DD_` prefix (example: `hostname` -> `DD_HOSTNAME`)
+ * Suboptions should be specified with an underscore (example: `DD_PROXY_HTTP`)
+ * List of values should be separated by spaces (example: `DD_AC_INCLUDE="image:cp-kafka image:k8szk"`)
+ * Any map structure should be json-formatted (example: `DD_DOCKER_ENV_AS_TAGS='{ "ENVVAR_NAME": "tag_name" }'`)
```

Comment:
Are we sure ? I don't think you can set the proxy setting with a JSON

Reply:
No it doesn't work for the proxy setting

Reply:
So we shouldn't say `Any map structure should be json-formatted` then.

Reply:
Yes actually it's any map structure but the proxy setting

Reply:
I've updated the docs, hopefully it is clearer